packages/iTwinUI-react/package.json

```diff
@@ -0,0 +1,147 @@
+{
+  "name": "@itwin/itwinui-react",
+  "version": "1.35.0",
+  "author": "Bentley Systems",
+  "license": "MIT",
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "typings": "cjs/index.d.ts",
+  "files": ["cjs", "esm"],
+  "description": "A react implementation of our iTwinUI UX standards",
+  "homepage": "https://github.com/iTwin/iTwinUI-react",
+  "keywords": [
+    "component",
+    "components",
+    "design-system",
+    "design",
+    "framework",
+    "frontend",
+    "itwin",
+    "itwinui",
+    "react-component",
+    "react",
+    "ui",
+    "ux"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn copy-files && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
+    "build:watch": "yarn clean && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
+    "clean": "yarn clean:coverage && rimraf esm && rimraf cjs",
+    "clean:coverage": "rimraf coverage",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "createComponent": "node scripts/createComponent.js",
+    "format": "prettier --config .prettierrc **/*.{tsx,ts,js} --ignore-path .gitignore --write",
+    "lint": "eslint --ext .js,.ts,.tsx --max-warnings=0",
+    "lint:fix": "yarn lint --fix && yarn lint:copyright --fix */**/*.{js,ts,tsx}",
+    "lint:copyright": "node scripts/copyrightLinter.js",
+    "copy-files": "cpx ../../README.md .",
+    "storybook": "start-storybook -p 6006",
+    "dev": "yarn storybook",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "@itwin/itwinui-css": "^0.53.3",
+    "@itwin/itwinui-icons-react": "^1.5.0",
+    "@itwin/itwinui-illustrations-react": "^1.0.1",
+    "@tippyjs/react": "^4.2.5",
```

Comment:
we have `4.2.6` currently in `package.json`

Reply:
Do we? https://github.com/iTwin/iTwinUI-react/blob/main/package.json#L46

Reply:
oh...
not sure where I have looked then :D